### PR TITLE
Issue 603 ev44 now

### PR DESCRIPTION
**Pull request by mortenjc**
### Issue reference / description

The branch you merge from should already reference an event-formation-unit github ticket number. You can add a descriptive title, but if an issue is referenced, you don't have to.

## Checklist for submitter

- [ ] Check for conflict with integration test
- [ ] Unit tests pass

---

### Nominate for Group Code Review

- [ ] Nominate for code review

### CODE REVIEW

File: src/common/CMakeLists.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ add_subdirectory(kafka)
 set(efu_common_SRC
   debug/Hexdump.cpp
   detector/EFUArgs.cpp
-  kafka/EV42Serializer.cpp
+  kafka/EV44Serializer.cpp
```
Comment (JennyLouise) [inline src/common/CMakeLists.txt:13]: repeated

File: src/modules/caen/CaenBase.h [hunk 2]
```diff
@@ -25,8 +25,8 @@ class CaenBase : public Detector {
 
 protected:
   struct Counters Counters;
-  EV42Serializer *Serializer;
-  EV42Serializer *SerializerII;
+  EV44Serializer *Serializer;
+  EV44Serializer *SerializerII;
```
Comment (JennyLouise) [inline src/modules/caen/CaenBase.h:29]: Do we need SerializerII?

File: src/modules/perfgen/PerfGenBase.cpp [hunk 3]
```diff
@@ -92,15 +92,15 @@ void PerfGenBase::processingThread() {
...
+    Serializer.checkAndSetReferenceTime(EfuTime);
 
     for (int i = 0; i < EventsPerPulse; i++) {
       auto PixelId = UdderImage.getPixel(ESSGeom.nx(), ESSGeom.ny(), &ESSGeom);
       Serializer.addEvent(TimeOfFlight, PixelId);
       mystats.events_udder++;
       TimeOfFlight++;
     }
-    // Serializer.checkAndSetPulseTime(EfuTime);
+    // Serializer.checkAndSetReferenceTime(EfuTime);
```
Comment (JennyLouise) [inline src/modules/perfgen/PerfGenBase.cpp:103]: remove

**Review by JennyLouise — APPROVED** [review]

File: src/modules/caen/CaenBase.h (hunk 2 above)
Comment (mortenjc) [reply src/modules/caen/CaenBase.h:29]: Probably not, it is implementing a 'debug' format requested by Davide. I'll leave it for now and make a ticket.

**Pull request merged**